lib/dpl/providers/pages/git.rb

```diff
@@ -96,7 +96,7 @@ def deploy
           git_clone? ? git_clone : git_init
           copy_files
           git_config
-          git_commit
+          git_commit if git_dirty?
           git_push
```

Comment:
Do we need to push if the local repo isn't dirty? In fact, if it's not dirty, do we need to bother setting up git?

Reply:
good points ... 🤔

Reply:
addressed, i think?